desktop/core/src/desktop/api2.py

```diff
@@ -850,7 +865,13 @@ def kill_task(request, task_id):
     return JsonResponse({'status': 'error', 'message': f'Failed to terminate task {task_id}: {str(e)}'})
 
 
-def get_task_logs(request, task_id):
+@api_error_handler
+@require_GET
+def get_task_logs(request):
+  task_id = request.GET.get('task_id')
```

Comment:
@Harshg999, can we refactor the code to use a function for retrieving task_id since it's being repeated in three places?